eZ/Bundle/EzPublishCoreBundle/Resources/config/default_settings.yml

```diff
@@ -30,6 +31,10 @@ parameters:
                 template: %ezplatform.default_view_templates.content.line%
                 match: []
         embed:
+            image:
+                template: '%ezplatform.default_view_templates.content.embed_image%'
+                match:
+                    IsEmbedImage: true
```

Comment:
>  if not for anything else, at least for readability and ability to copy and modify if you have another type.

the idea is actually to not have to copy paste anything and to avoid trying to guess whether the embed is an image or not since The markup provides that info.
IMHO, this is easier and cleaner than trying to guess whether the editor wanted to have an image or not.